### PR TITLE
Add Starknet Firehose setup pages

### PR DESCRIPTION
**Pull request by xJonathanLEI**
Adds a `Starknet` section under the `Firehose Setup`.

### CODE REVIEW

**Comment by xJonathanLEI** [comment]
Deployed preview available at: https://xjonathanlei.gitbook.io/firehose/firehose-setup/starknet

File: SUMMARY.md [hunk 1]
```diff
@@ -31,6 +31,10 @@
   * [Single-Machine Deployment](firehose-setup/cosmos/local-deployment.md)
 * [NEAR](firehose-setup/near/README.md)
   * [Single-Machine Deployment](firehose-setup/near/installation.md)
+* [Starknet](firehose-setup/starknet/README.md)
```
Comment (enoldev) [inline SUMMARY.md:34]: Could you create a new section under `Firehose Setup` called `Community Integrations` and add the Starknet under it? We're trying to separate the chains we actually maintain and support.

I can also do it if you want to

Comment (xJonathanLEI) [reply SUMMARY.md:34]: Sure lemme move everything over.

**Comment by xJonathanLEI** [comment]
@enoldev Moved the subsection to a new section as instructed.

**Review by enoldev — APPROVED** [review]

**Pull request merged**